images/linux/scripts/tests/Databases.Tests.ps1

```diff
@@ -3,7 +3,8 @@ Describe "MongoDB" {
         @{ ToolName = "mongo" }
         @{ ToolName = "mongod" }
     ) {
-        "$ToolName --version" | Should -ReturnZeroExitCode
```

Comment:
I think it makes sense to leave this test as well

Reply:
As discussed keeping this is redundant